webapp/display/field_group.js

```diff
@@ -94,7 +95,7 @@ export var DiffFormMixin = {
   }
 }
 
-export var create = function(form, saveButtonText, _this, messages=[], extraButtons=[]) {
+export var createForm = function(form, saveButtonText, _this, messages=[], extraButtons=[]) {
```

Comment:
I would separate this create -> createForm change into a separate revision. Since the imports changes are entirely mechanical (outside of the write_includes.py changes), it's probably clearest to keep additional manual changes separate.